packages/core/src/utils/match.js

```diff
@@ -1,7 +1,17 @@
-export default function match(url, pattern) {
-  if (pattern === undefined) {
+import isArray from 'lodash/isArray';
+import isString from 'lodash/isString';
+import map from 'lodash/map';
+import some from 'lodash/some';
+
+export default function match(url, patternOrArray) {
+  if (patternOrArray === undefined) {
```

Comment:
maybe the pattern to `always match` should be asterisk?

Reply:
look at the tests - the api for adding a url that applies to all urls is super obvious
`request.use((req, res, next) => next())`